src/omotes_orchestrator/main.py

```diff
@@ -396,6 +401,40 @@ def job_cancellation_handler(self, job_cancellation: JobCancel) -> None:
                 )
                 self._cleanup_job(job_id)
 
+    def dead_lettered_job_result_handler(self, job_result: JobResult) -> None:
+        """Handle the received dead lettered job result.
+
+        When the log level is set at the DEBUG level, the dead lettered job result will be written
+        to a local file.
+
+        :param job_result: Job result message.
+        """
+        logger.info("Received a dead lettered job_%s_result with result type as: %s",
+                    job_result.uuid,
+                    job_result.result_type)
+
+        log_level = logger.getEffectiveLevel()
+        if log_level == logging.DEBUG:
+            try:
+                log_dir = "../logs"
```

Comment:
This path should be configurable

Reply:
Updated in 1a5bb3fe40a03f272badacea7bd8e413a3c8c4a5

I now make it as an Orchestrator class attribute as I don't feel it is needed to set this in `.env` file. If there is a better way of doing it, let me know!